Assessment.Test/Assessment.Test/ViewModels/LoginViewModel.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Assessment.Test.Models;
+using Assessment.Test.ViewModels.Base;
+using Assessment.Test.Views;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Text;
+using Xamarin.Forms;
+
+namespace Assessment.Test.ViewModels
+{
+    public class LoginViewModel : INotifyPropertyChanged
```

Comment:
Inherit this class from base class.
Base class is inherited from INotifyPropertyChanged.
More helpful  implementation are in base class.So inherit all the view model from base class